src/main/java/school/lemon/changerequest/java/vehicles/Airplane.java

```diff
@@ -0,0 +1,58 @@
+package school.lemon.changerequest.java.vehicles;
+
+/**
+ * Created by akliuieva on 29.06.17.
+ */
+public class Airplane extends Vehicle implements IAirplane{
+    private int maximumHeightFeet;
+
+    public Airplane(int manufacturedYear, String make, String model, int maximumHeightFeet) {
+        super(manufacturedYear, make, model);
+        this.maximumHeightFeet = maximumHeightFeet;
+    }
+
+    @Override
+    public int getMaximumHeightFeet() {
+        return maximumHeightFeet;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Airplane airplane = (Airplane) o;
+
+        return Math.abs(maximumHeightFeet - airplane.maximumHeightFeet) <= 1000;
+    }
+
+    @Override
+    public String toString() {
+        return "This airplane is a " + getManufacturedYear() + " " + getMake() + " " + getModel() + " that can reach "+getMaximumHeightFeet()+" feet.";
+    }
+
+    @Override
+    public int hashCode() {
```

Comment:
Do you remember equals+hashCode contract? 
If objects are equal - hash code should be the same.